pallets/simple-staking/src/lib.rs

```diff
@@ -0,0 +1,162 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+/// Edit this file to define custom logic or remove it if it is not needed.
+/// Learn more about FRAME and the core library of Substrate FRAME pallets:
+/// <https://substrate.dev/docs/en/knowledgebase/runtime/frame>
+pub use pallet::*;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
```suggestion
#[cfg(any(feature = "runtime-benchmarks", test))]
```

Will be helpful to run benchmark tests as part of main tests as well, at least during development